web/business-registry-dashboard/devops/vaults.env

```diff
@@ -1,26 +1,37 @@
-# registries search api
+# NOTE! nuxt-core-layer-test requires the following keys:
+#    NUXT_KEYCLOAK_AUTH_URL
+#    NUXT_KEYCLOAK_REALM
+#    NUXT_KEYCLOAK_CLIENTID
+#    NUXT_AUTH_WEB_URL
+#    NUXT_AUTH_API_VERSION
+#    NUXT_AUTH_API_URL
```

Comment:
This is the one that THIS UI doesn't use (so won't show up in global search) but we need it declared or things will fail.